enhancements/machine-config/machine-config-node.md

```diff
@@ -0,0 +1,340 @@
+
+```console
+$ oc get machineconfignodes
+NAME                          UPDATED   UPDATEPREPARING   UPDATEINPROGRESS   UPDATEPOSTACTION   UPDATECOMPLETING   RESUMING
+ip-10-0-12-194.ec2.internal   True      False             False              False              False              False
+ip-10-0-17-102.ec2.internal   True      False             False              False              False              False
+ip-10-0-2-232.ec2.internal    True      False             False              False              False              False
+ip-10-0-59-251.ec2.internal   True      False             False              False              False              False
+ip-10-0-59-56.ec2.internal    True      False             False              False              False              False
+ip-10-0-6-214.ec2.internal    True      False             False              False              False              False
+```
+
+as well as 
+
+```console
+$ oc get machineconfignodes -o wide
+NAME                          UPDATED   UPDATEPREPARING   UPDATEINPROGRESS   UPDATEPOSTACTION   UPDATECOMPLETING   RESUMING   COMPARINGMACHINECONFIGS   UPDATINGFILESANDOS   DRAININGNODE   CORDONINGNODE   REBOOTINGNODE   RELOADINGCRIO
+ip-10-0-1-37.ec2.internal     True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-33-243.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-37-208.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-39-225.ec2.internal   False     False             False              True               False              False      False                     False                False          False           True            False
+ip-10-0-51-162.ec2.internal   True      False             False              False              False              False      False                     False                False          False           False           False
+ip-10-0-59-150.ec2.internal   False     False             False              True               False              False      False                     False                False          False           True            False
+```
+
+Where each name represents a node in a given machineconfigpool. The statuses reported are created explicitly from MCO node annotations and MCO actions, no other operator actions are taken into account here. This allows us to get quite specific in what is occuring on the nodes.
+
+```console
+Name:         ip-10-0-12-194.ec2.internal
+Namespace:    
+Labels:       <none>
```

Comment:
Should these be labeled by pool, so you can conveniently `oc get machineconfignodes -l something="${MY_POOL_NAME}" -o wide`?

Reply:
We are putting it in the spec, so you could field selector on it, but a label would also be helpful

Reply:
fieldSelector in 4.15/16 is the resolution here. Unless there is a strong opinion over adding a label, I think this is fine for most use cases